### DISCUSSION TITLE
if panel:getBootstrapState() then return

### DISCUSSION THREAD

**Discussion by DRBAUS** [RomanKubiak/ctrlr#465] [Q&A] [answered]
What does this mean and what does it do?

**Comment by Godlike-Productions** [comment, marked as answer]
We use it to protect LUA from crashing if some code is trying to run during initialization of a panel.  For example, trying to assign a modulator value to a modulator that is not yet present.

Basically it's a check that ctrlr is in a state that it can operate in

**Reply by DRBAUS** [reply]
Thanks!

I was already wondering how that would work: assign a modulator value before modulators are created.

It makes sense now.
